### ISSUE TITLE
changing the package name or version

### ISSUE THREAD

**Issue by Spp595** [MatsuriDayo/NekoBoxForAndroid#660]
The Google Play version is no longer official. Maybe you should change the names of the github package so that google play cannot update itself to the "wrong" version?

**Comment by Naushikha** [comment]
Yes I second this, the app automatically updates itself to MiProxy VPN
